### PR TITLE
Applied JuliaFormatter to all of Javis.jl files

### PR DESCRIPTION
**Pull request by TheCedarPrince**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [x] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [x] Did I update the `TODO.md` (if applicable)?
- [x] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [x] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [x] Did I properly add test dependencies to the `test` directory (if applicable)?
- [x] Did I check relevant tutorials that may be affected by changes in this PR?
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**Link to relevant issue(s)**

#123 

**How did you address these issues with this PR? What methods did you use?**

I ran `JuliaFormatter.jl`'s `format(".")` function to recursively format each file to show how the formatting looks like. Though this PR would not fix everything for making a GitHub Action, it will allow a point for each piece of code to be formatted via JuliaFormatter from which we could make a GitHub Action.

Further, each piece of our code from this point forward could be in compliance with JuliaFormatter. 

What do you think @Wikunia ?

### CODE REVIEW

File: src/morphs.jl [hunk 1]
```diff
@@ -135,10 +135,10 @@ function save_morph_polygons!(action::Action, from_func::Function, to_func::Func
     smallest_i = 1
     smallest_distance = typemax(Float64)
 
-    for i in 1:length(from_poly)
+    for i = 1:length(from_poly)
         overall_distance = 0.0
         for j = 1:length(from_poly)
-            p1 = from_poly[(j+i-1) % length(from_poly) + 1]
+            p1 = from_poly[(j+i-1)%length(from_poly)+1]
```
Comment (Wikunia) [inline src/morphs.jl:141]: I actually hate that there is no space around the %. 😄  Any idea why? Are there options to change that behavior?

File: src/morphs.jl [hunk 2]
```diff
@@ -148,8 +148,8 @@ function save_morph_polygons!(action::Action, from_func::Function, to_func::Func
         end
     end
     new_from_poly = copy(from_poly)
-    for i in 1:length(from_poly)
-        new_from_poly[i] = from_poly[(i+smallest_i-1) % length(from_poly) + 1]
+    for i = 1:length(from_poly)
+        new_from_poly[i] = from_poly[(i+smallest_i-1)%length(from_poly)+1]
```
Comment (Wikunia) [inline src/morphs.jl:152]: I also prefer `in` as `=` should be for an assignment in my opinion

**Review by Wikunia — COMMENTED** [review]
Some things I would like to have changed if possible. If not possible I'm not sure whether I'm in favor of using JuliaFormatter to be honest.

File: src/morphs.jl (hunk 2 above)
Comment (TheCedarPrince) [reply src/morphs.jl:152]: I agree! There is a way to fix this using the kwarg, `always_for_in` set to true when formatting.

File: src/morphs.jl (hunk 1 above)
Comment (TheCedarPrince) [reply src/morphs.jl:141]: I don't know why, but there is a way to disable this using the kwarg, `whitespace_ops_in_indices` set to true.

**Comment by TheCedarPrince** [comment]
Hey @Wikunia could you tell me what else you do/do not like to see if JuliaFormatter could take care of it for us? I made your changes that you first saw were wrong and fixed those correctly.

**Review by Wikunia — COMMENTED** [review]
LGTM.
Shall we merge this once you have added it to GitHub Actions. I'm thinking how to make the least number of merge conflicts.

**Comment by TheCedarPrince** [comment]
I think we could. So the way I was thinking this could be correctly implemented with this PR:

1. Roll back any changes on this branch
2. When you are done with any PRs you want to merge, let me know
3. Merge this branch with master so that it is even
4. After I make this branch equal with master, let you know that I am going to update the GitHub CI with JuliaFormatter
5. Merge to master.
6. Check that all tests pass on master and nothing was broken

From there, any branch that we have can then be made compatible with JuliaFormatter. Before I continue on this, could you double check the [documentation about any particulars you may want to have modified](https://domluna.github.io/JuliaFormatter.jl/stable/api/#JuliaFormatter.format_text-Tuple{AbstractString})? Meaning, kwargs to be activated or disabled, specific formatting, etc. We could even [define our own behaviors](https://domluna.github.io/JuliaFormatter.jl/stable/custom_styles/) if we wanted to.

Here is the command I am currently using to modify all files in a directory:

```julia
format(".", 
annotate_untyped_fields_with_any=false, 
always_for_in=true, 
whitespace_in_kwargs=true, 
whitespace_ops_in_indices=true)
```

What are your thoughts @Wikunia ?

**Comment by Wikunia** [comment]
Sounds good. I had a look at the modifiers earlier. I think the ones you applied are the ones I would use. I would like to not indent `@testset` inside `@testset` but that's probably just me and not worth going in to. 
You can merge the documentation PR which is not linked any way and then do this one. When exactly is the formatting applied? Inside the PR, right? I think merge conflicts might be handled correctly by formatting the PR I work on before I push and later there shouldn't be a problem.

**Comment by TheCedarPrince** [comment]
I think that this is ready for merging now @Wikunia . What do you think? I created the GitHub Action that will run JuliaFormatter anytime a new push is made to master.

**Comment by Wikunia** [comment]
Do you want to revert all changes other than the GitHub Action to see whether this actually works?

**Pull request merged**